python-package/securexgboost/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 import os
 
-from .core import DMatrix, Booster, Enclave, CryptoUtils, set_user
+from .core import DMatrix, Booster, Enclave, CryptoUtils, Set_user, User
```

Comment:
I think we can remove `Set_user` here if it's in the User class.

Reply:
deleted